lib/routers/establishment/index.js

```diff
@@ -61,6 +61,7 @@ router.use('/:establishment/place(s)?', require('./places'));
 router.use('/:establishment/profile(s)?', require('../profile'));
 
 router.use('/:establishment/project(s)?', require('./projects'));
+router.use('/:establishment/project-version(s)?', require('./project-versions'));
```

Comment:
Is it possible to scope this underneath a particular project?

i.e. `/establishment/:id/project(s)/:id/version(s)/:id`

Reply:
On the basis that `GET /establishment/:id/project/:id/versions` is highly likely to be a thing we want.

Reply:
yeah, I was going to do both, but I'm not sure if establishment/id/versions is actually a thing

Reply:
I'll update it

Reply:
Yeah, I think because a version can only exist within the scope of a parent project then it makes sense to build the endpoints that way.